test/basic.test.js

```diff
@@ -155,9 +159,9 @@ test('Will message', async function (t) {
 
   await client2.subscribe('my/will', { qos: 1 })
 
-  client.stream.destroy()
+  client._client.stream.destroy()
```

Comment:
It's nested because I'm using `async-mqtt` and `_client` is the original client object (with no promises) and it also expose the stream param that in this case is used to force close the steam so the broker will send the will, do you have any suggestion how I could end the client connection so the broker will detect an unexpected disconnect?

Reply:
Maybe just add a comment on top.